src/libs/OptionsListUtils.ts

```diff
@@ -534,10 +534,14 @@ function uniqFast(items: string[]): string[] {
  * Get the last actor display name from last actor details.
  */
 function getLastActorDisplayName(lastActorDetails: Partial<PersonalDetails> | null, hasMultipleParticipants: boolean) {
-    return hasMultipleParticipants && lastActorDetails && lastActorDetails.accountID !== currentUserAccountID
+    if (!hasMultipleParticipants || !lastActorDetails) {
```

Comment:
@nkdengineer Can you please add unit test for this method to cover all its paths? Thanks!